api-service/webapp-demo/src/app/crowd_source_api.service.ts

```diff
@@ -0,0 +1,83 @@
+/*
+Copyright 2017 Google Inc.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { Observable } from 'rxjs/Observable';
+import 'rxjs/add/observable/fromPromise';
+import 'rxjs/add/operator/map';
+
+export interface WorkToDo {
```

Comment:
Maybe you can add a comment that I should have added before: that because these structures come from the DB, they currently have underscore word separaters instead of useing CamelCase. TODO make the DB, and field names, and any uses in the server, also be CamelCase so that we can use a more consistent code style.